src/catalog/src/builtin.rs

```diff
@@ -7281,6 +7330,19 @@ pub const MZ_SUPPORT_CLUSTER: BuiltinCluster = BuiltinCluster {
     ],
 };
 
+pub const MZ_ANALYTICS_CLUSTER: BuiltinCluster = BuiltinCluster {
+    name: "mz_analytics",
+    owner_id: &MZ_ANALYTICS_ROLE_ID,
+    privileges: &[
+        MzAclItem {
+            grantee: MZ_SYSTEM_ROLE_ID,
+            grantor: MZ_ANALYTICS_ROLE_ID,
+            acl_mode: rbac::all_object_privileges(SystemObjectType::Object(ObjectType::Cluster)),
```

Comment:
@jkosh44 this won't be enough for `mz_analytics` to alter this cluster, right? Because it's still owned by `mz_system`? 

Ditto for the builtin connection added elsewhere.

Reply:
I might not be understanding something, but `mz_analytics` is listed as the owner, not `mz_system`. The owner (`mz_analytics`) can alter the cluster. The connection is owned by `mz_system`, so `mz_analytics` cannot alter it.

Reply:
Ah, sorry, I got a bit turned around somewhere. The first revision didn't have an `owner_id` field on `BuiltinConnection` so the connection was created as owned by `mz_system`. Everything looks good now.